src/analysis/sum_integral_comparisons.lean

```diff
@@ -0,0 +1,87 @@
+/-
+Copyright (c) 2022 Kevin H. Wilson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin H. Wilson
+-/
+import measure_theory.measure.measure_space_def
+import measure_theory.integral.interval_integral
+import algebra.order.floor
+import analysis.special_functions.integrals
+
+/-!
+# Comparing sums and integrals
+
+## Summary
+
+It is often the case that error terms in analysis can be computed by converting the
+an infinite sum to the improper integral of an antitone function. This file enables that.
```

Comment:
Currently, the docstring is not accurate as the file only contains the case of finite intervals. Can you adjust the docstring to the current state of the file?

Reply:
Sure!